.github/workflows/pwsh.yaml

```diff
@@ -12,14 +12,16 @@ jobs:
 
   test:
     name: Test Generated PowerShell Cmdlets
-    runs-on: windows-latest
+    strategy:
+      matrix:
+        ruby: [2.7, 3.1]
```

Comment:
The changes in this file shouldn't be necessary since it's just testing the generated cmdlets.